port/ztpf/omrosdump_helpers.c

```diff
@@ -851,7 +848,8 @@ buildELFNoteArea(Elf64_Nhdr *buffer, DIB *dibPtr)
 	 * fractional. Does it really matter? Nah. But if you're gonna do it, you might
 	 * as well do it right ....
 	 */
-	prstat->stime.tv_usec = ecbptr()->ce1istim / 4098; /* Used CPU time    */
+	/* prstat->stime.tv_usec = ecbptr()->ce1istim / 4098; Used CPU time    */
```

Comment:
Is the commented out line valuable?  If so can you document why we are keeping it around?  Thanks